src/common/localize.ts

```diff
@@ -89,6 +89,12 @@ export namespace VenvManagerStrings {
 
     export const installEditable = l10n.t('Install project as editable');
     export const searchingDependencies = l10n.t('Searching for dependencies');
+
+    export const selectQuickOrCustomize = l10n.t('Select Quick Create or Customize');
```

Comment:
Not sure if "mode" is the best in this context
```suggestion
    export const selectQuickOrCustomize = l10n.t('Select environment creation mode');
```